.github/workflows/rust.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: audit
-          args: --deny-warnings --ignore RUSTSEC-2019-0011 --ignore RUSTSEC-2019-0013 --ignore RUSTSEC-2020-0016
+          args: --deny-warnings --ignore RUSTSEC-2019-0011 --ignore RUSTSEC-2019-0013 --ignore RUSTSEC-2020-0016 --ignore RUSTSEC-2020-0036
```

Comment:
YAML has a multiline syntax that helps for cases this. Example: https://github.com/RustSec/cargo-audit/pull/256/files